automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -89,4 +107,16 @@ object Leonardo extends WorkbenchClient with LazyLogging {
     }
 
   }
+
+  object notebooks {
+    def notebooksPath(googleProject: GoogleProject, clusterName: ClusterName): String =
+      s"notebooks/${googleProject.string}/${clusterName.string}"
+
+    def get(googleProject: GoogleProject, clusterName: ClusterName)(implicit token: AuthToken, webDriver: WebDriver): NotebooksListPage = {
```

Comment:
As an alternative, `NotebooksListPage` can be responsible for knowing how to construct its URL from a `GoogleProject` and `ClusterName`.